### PR TITLE
Update Hardcoded Kafka Properties to be Configurable

### PR DESCRIPTION
**Pull request by mwodahl**
Update to allow users to set the kafka auto commit, auto commit interval, and session timeout values as environment variables.

### CODE REVIEW

File: sample.env [hunk 1]
```diff
@@ -14,6 +14,9 @@ HEADER_X_API_KEY=
 KAFKA_TYPE=
 CONFLUENT_KEY=
 CONFLUENT_SECRET=
+KAFKA_ENABLE_AUTO_COMMIT=
+KAFKA_AUTO_COMMIT_INTERVAL_MS=
+KAFKA_SESSION_TIMEOUT_MS=
```
Comment (dmccoystephenson) [inline sample.env:19]: It might be good to put a comment in here saying that if these are not set, they will have default values.

File: src/main/java/us/dot/its/jpo/ode/aws/depositor/AwsDepositor.java [hunk 2]
```diff
@@ -176,7 +185,7 @@ public void run(String[] args) throws Exception {
 				boolean gotMessages = false;
 
 				while (true) {
-					ConsumerRecords<String, String> records = stringConsumer.poll(CONSUMER_POLL_TIMEOUT_MS);
+					ConsumerRecords<String, String> records = stringConsumer.poll(Duration.ofMillis(CONSUMER_POLL_TIMEOUT_MS));
```
Comment (dmccoystephenson) [inline src/main/java/us/dot/its/jpo/ode/aws/depositor/AwsDepositor.java:188]: Why was this necessary?

**Review by dmccoystephenson — CHANGES_REQUESTED** [review]
Looks good, just a couple comments

File: src/main/java/us/dot/its/jpo/ode/aws/depositor/AwsDepositor.java (hunk 2 above)
Comment (mwodahl) [reply src/main/java/us/dot/its/jpo/ode/aws/depositor/AwsDepositor.java:188]: The poll(long) method was deprecated - I updated this to use a duration instead so that there would no longer be a deprecation warning.

File: sample.env (hunk 1 above)
Comment (mwodahl) [reply sample.env]: Updated this to include comments with the default values for each of these variables.

Comment (dmccoystephenson) [reply sample.env]: I'm unsure whether comments are allowed on the same line as environment variables in .env files. Developers might need to delete the comments to set the environment variables correctly. It's probably better to have these comments on separate lines from the environment variables.

Comment (mwodahl) [reply sample.env]: Updated to have comments on separate lines

**Review by dmccoystephenson — APPROVED** [review]
Looks good to me!

**Review by drewjj — APPROVED** [review]
Looks good!
